Gruntfile.js

```diff
@@ -195,37 +202,30 @@ module.exports = function( grunt ) {
 	} );
 
 	grunt.registerTask( "lint", [
-		"jsonlint",
-		runIfNewNode( "eslint:dev" ),
-		runIfNewNode( "eslint:dist" )
+		"newer:jsonlint",
+		runIfNewNode( "newer:eslint:dev" ),
+		runIfNewNode( "newer:eslint:dist" )
 	] );
 
-	grunt.registerTask( "test_fast", [ runIfNewNode( "node_smoke_tests" ) ] );
```

Comment:
> Would like to land this tomorrow

Is there a reason to hurry? This PR is pretty isolated, even if it were to land in a few weeks, rebasing it would be trivial.